pytest_order/plugin.py

```diff
@@ -36,12 +36,16 @@ def pytest_configure(config: Config) -> None:
         # to keep the function as a function and then add it to the class as a
         # pseudo method.  Since the class is purely for structuring and `self`
         # is never referenced, this seems reasonable.
```

Comment:
The Python 2.7 related part of the comment can now be removed.

Reply:
Hmm, reading the comment now, it may be possible to put the `modify_items` method back inside the `OrderingPlugin` class. The if statement for indulgent ordering could be moved inside the method, and put inside a dynamic decorator, which will add the hook implementation with either `tryfirst` or `trylast`, based on the value returned. The `config` var it accesses, could be passed through the `__init__`.

I'll prepare something like this, for you to review in the next commit.

Reply:
Yes, I actually thought about that too, but didn't get to it for some reason... would be nice.